From e0cc9c5c4093764ad76f85dd40698825f3a8f91d Mon Sep 17 00:00:00 2001
From: "Valery V. Vorotyntsev" <valery_vorotyntsev@xyratex.com>
Date: Sat, 9 Mar 2013 22:26:54 +0200
Subject: [PATCH 10/24] rpc/rpclib.h: don't use doxygen where it's not
 applicable

Make "input" and "output" portions of m0_rpc_client_ctx parameters
visually distinguishable.  Original doxygenic solution was wrong,
it could not render correctly.
---
 rpc/rpclib.h | 11 ++++-------
 1 file changed, 4 insertions(+), 7 deletions(-)

diff --git a/rpc/rpclib.h b/rpc/rpclib.h
index db42dcb..1231f04 100644
--- a/rpc/rpclib.h
+++ b/rpc/rpclib.h
@@ -49,7 +49,6 @@ struct m0_reqh_service_type;
  * using mero-setup API.
  */
 struct m0_rpc_server_ctx {
-
 	/** a pointer to array of transports, which can be used by server */
 	struct m0_net_xprt          **rsx_xprts;
 	/** number of transports in array */
@@ -107,12 +106,10 @@ m0_rpc_server_ctx_get_rmachine(struct m0_rpc_server_ctx *sctx);
  * Contains all required data to initialize an RPC client and connect to server.
  */
 struct m0_rpc_client_ctx {
-
-	/**
+	/*
 	 * Input parameters.
 	 *
-	 * They are initialized and filled in by a caller of
-	 * m0_rpc_client_start().
+	 * They are initialised and filled by a caller of m0_rpc_client_start().
 	 */
 
 	/**
@@ -156,10 +153,10 @@ struct m0_rpc_client_ctx {
 	 */
 	uint32_t		   rcx_timeout_s;
 
-	/**
+	/* -------------------------------------------------------------
 	 * Output parameters.
 	 *
-	 * They are initialized and filled in by m0_rpc_client_start().
+	 * They are initialised and filled by m0_rpc_client_start().
 	 */
 
 	struct m0_rpc_machine	   rcx_rpc_machine;
-- 
1.8.3.2

